pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -568,6 +568,27 @@
             "#import": [ "opensuse.42.1", "opensuse-x64" ]
         },
 
+        "gentoo": {
+            "#import": [ "linux" ]
+        },
+        "gentoo-x64": {
+            "#import": [ "gentoo", "linux-x64" ]
+        },
+
+        "buildroot": {
+            "#import": [ "linux" ]
+        },
+        "buildroot-x64": {
+            "#import": [ "buildroot", "linux-x64" ]
+        },
+
+        "buildroot.2017.02.2": {
+            "#import": [ "buildroot" ]
+        },
+        "buildroot.2017.02.2-x64": {
```

Comment:
If Buildroot-x64 can be used, that would be the best solution I think.

Would that work, when VersionId has a value defined in /etc/os-release?